pyccel/parser/syntactic.py

```diff
@@ -172,10 +176,11 @@ def __init__(self, inputs, **kwargs):
                           severity='fatal')
 
         preprocess_imports(red)
-        preprocess_default_args(red)
+        #preprocess_default_args(red)
```

Comment:
Is this broken or handled elsewhere now?

Reply:
it's handled elsewhere in Functioncall and print_Functioncall

Reply:
Then can we get rid of this line and the associated function?

Reply:
This line has been removed however the function itself is still in parser/utilities.py. Is it used elsewhere? I didn't think it was. If not then we should remove the definition too

Reply:
done